### PR TITLE
Issue 1574 submit local runs missing fields

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Review by eamansour — APPROVED** [review]

File: pkg/runs/submitter.go [hunk 1]
```diff
@@ -465,13 +465,15 @@ func (submitter *Submitter) isRasDetailNeededForReports(params utils.RunsSubmitC
 
 func (submitter *Submitter) buildListOfRunsToSubmit(portfolio *Portfolio, runOverrides map[string]string) []TestRun {
 	readyRuns := make([]TestRun, 0, len(portfolio.Classes))
-
+	currentUser , _ := user.Current()
```
Comment (eamansour) [inline pkg/runs/submitter.go:468]: As discussed, we can use the `GetCurrentUserName()` method here instead since that performs some error handling.

**Review by eamansour — CHANGES_REQUESTED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-157-cj98f for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/runs/submitter.go [hunk 2]
```diff
@@ -529,10 +531,7 @@ func (submitter *Submitter) validateAndCorrectParams(
 	}
 
 	if err == nil {
-		if params.Requestor == "" {
-			// Requestor has not been set. Default it to the current user id.
-			params.Requestor, err = submitter.env.GetUserName()
-		}
+		params.Requestor, err = submitter.env.GetUserName()
```
Comment (eamansour) [inline pkg/runs/submitter.go:534]: If the requestor will always be the current user, I don't think we'd need the requestor in the `RunsSubmitCmdParameters` struct anymore (which also means removing it from the available flags passed in from the user) - is this intentional?

Comment (Wyvinar) [reply pkg/runs/submitter.go:534]: This part should be code that was already there, and would be changed in a new issue that we are currently working on.

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-157-s2b4r for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by KirbyKatcher — CHANGES_REQUESTED** [review]
Unit tests are failing

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-157-jp2qx for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Comment by Akyiaa** [comment]
Merged changes from issue [1570](https://github.com/galasa-dev/projectmanagement/issues/1570) to remove the requestor flag and related tests from cli side

**Review by KirbyKatcher — APPROVED** [review]

**Pull request merged**